airbyte-integrations/connectors/source-trustpilot/metadata.yaml

```diff
@@ -2,7 +2,9 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: d7e23ea6-d741-4314-9209-a33c91a2e945
-  dockerImageTag: 0.1.0
+  connectorBuildOptions:
+    baseImage: docker.io/airbyte/python-connector-base:1.1.0@sha256:bd98f6505c6764b1b5f99d3aedc23dfc9e9af631a62533f60eb32b1d3dbab20c
```

Comment:
There is a newer version available, 1.2.0 — can you please lookup it's SHA code and replace this with it? 

A PR to the migration tool is also welcome if you'd feel like exploring `airbyte-ci`.

Reply:
Done!